src/components/Publish/_validation.ts

```diff
@@ -54,13 +56,23 @@ const validationService = {
   })
 }
 
+const maxDecimalsValidation = new RegExp(/^\d+(\.\d{1,5})?$/)
```

Comment:
uuuh fun, does interpolation work in regex? Cause this also needs to be based on `MAX_DECIMALS`

Reply:
I tried that and it seems not, because of the non-literal argument found in the regex. looking for alternatives